examples/react-dapp/src/data/icrc49_call_canister.tsx

```diff
@@ -44,7 +44,14 @@ export const icrc49CallCanisterSection: ISection = {
   getCodeSnippet: function (requestJSON: string): string {
     const basicRequest = JSON.parse(requestJSON)
 
-    return `await IdentityKit.init()
-const response = await IdentityKit.request(${JSON.stringify(basicRequest, null, 2)})`
+    return `const { IdentityKitAgent } = useIdentityKit()
+const agent = new IdentityKitAgent({
+  getPrincipal: () => Principal.fromText("${basicRequest.params.sender}"),
```

Comment:
It'd be much better and useful for end developers to be able to pass `SignerAgentOptions` into `IdentityKitAgent`.